Sources/AppStoreConnectCLI/Model/BetaGroup.swift

```diff
@@ -62,6 +65,37 @@ extension BetaGroup {
     }
 }
 
+extension BetaGroup: Equatable, Hashable {
+
+    static func == (lhs: BetaGroup, rhs: BetaGroup) -> Bool {
+        return lhs.id == rhs.id &&
+            lhs.groupName == rhs.groupName &&
+            lhs.publicLinkEnabled == rhs.publicLinkEnabled &&
+            lhs.publicLinkLimit == rhs.publicLinkLimit &&
+            lhs.publicLinkLimitEnabled == rhs.publicLinkLimitEnabled
+    }
+
+    func hash(into hasher: inout Hasher) {
+        hasher.combine(id)
+        hasher.combine(groupName)
+        hasher.combine(publicLinkEnabled)
+        hasher.combine(publicLinkLimit)
+        hasher.combine(publicLinkLimitEnabled)
+    }
+}
+
+extension BetaGroup: SyncResultRenderable {
+    var syncResultText: String {
+        self.groupName ?? "group-name"
```

Comment:
I feel like if the group has no name we have a serious problem. Maybe we need to tighten our model and errors for not having a name